src/sentry/relay/projectconfig_debounce_cache/base.py

```diff
@@ -22,14 +22,23 @@ def __init__(self, **options):
 
     def check_is_debounced(self, public_key, project_id, organization_id):
         """
-        Check if the given project/organization should be debounced.
+        Check if the given project/organization should be debounced, and
+        debounces when it isn't.
 
         It's fine to erroneously return false, it's not fine to erroneously
         return true.
         """
 
         return False
 
+    def is_debounced(self, public_key, project_id, organization_id):
+        """Checks if the given project/organization should be debounced."""
+
+    def debounce(self, public_key, project_id, organization_id):
+        """
+        Debounces the given project/organization, without performing any checks.
+        """
```

Comment:
In the implementation, you return a boolean from `debounce`, could you describe the return value here?

Aside, this seems like a misleading and dangerous method to call from the outside since it does not actually debounce, it just increases the timeout of some existing key. Are you sure you need this in the public interface?

Reply:
> it just increases the timeout of some existing key
I think `setex` writes to the key regardless of whether it previously existed. https://redis.io/commands/setex/

IMO `debounce` should not return anything.